mythx_cli/analyze/truffle.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, target: Path):
         self.dependency_map = self.build_dependency_map()
 
     def find_truffle_artifacts(
-        self
+        self,
```

Comment:
Is this change caused by tweak in linter rules?

Reply:
probably, removed it :)